doc/manual/faq.rst

```diff
@@ -1,110 +1,171 @@
 
-organize experiment windows in the dashboard?
----------------------------------------------
+change configuration settings of satellite devices?
+---------------------------------------------------
 
-Experiment windows can be organized by using the following hotkeys:
+Currently, it is not possible to reach satellites through ``artiq_coremgmt config``, although this is being worked on. On Kasli, use :class:`~artiq.frontend.artiq_mkfs` and :class:`~artiq.frontend.artiq_flash`; on Kasli-SoC, preload the SD card with a ``config.txt``, formatted as a list of ``key=value`` pairs, one per line.
 
-* CTRL+SHIFT+T to tile experiment windows
-* CTRL+SHIFT+C to cascade experiment windows
+Don't worry about individually flashing idle or startup kernels. If your idle or startup kernel contains subkernels, it will automatically compile as a ``.tar``, which you only need to flash to the master.
 
-The windows will be organized in the order they were last interacted with.
+.. _faq-networking:
 
-write a generator feeding a kernel feeding an analyze function?
----------------------------------------------------------------
+troubleshoot networking problems?
+---------------------------------
 
-  Like this::
+Diagnosis aids:
 
-    def run(self):
-        self.parse(self.pipe(iter(range(10))))
+    - Can you ``ping`` the device?
+    - Is the Ethernet LED on?
+    - Is the ERROR LED on?
+    - Is there anything unusual recorded in :ref:`the UART log <connecting-UART>`?
+
+Some things to consider:
+
+    - Is the ``core_addr`` field of your ``device_db.py`` set correctly?
+    - Are your core device's IP address and networking configurations definitely set correctly? Check the UART log for evidence of this, and ask your network administrator about the other settings.
+    - Is your core device configured for an external reference clock? If so, it cannot function correctly without one. Is the external reference clock plugged in?
+    - Are SFP0 and Ethernet plugged in all the way? Are they working? Try different cables and SFP adapters; M-Labs tests with CAT6 cables, but lower categories should be supported too.
+    - Are your PC and your crate in the same subnet?
+    - Is some other device in your network already using the configured IP address? Turn off the core device and try pinging the configured IP address; if it responds, you have a culprit. One of the two will need a different networking configuration.
+    - Are there restrictions or issues in your router or subnet that are preventing the core device from connecting? It may help to try connecting the core device to your PC directly.
+
+Command-line tools like ``nmap`` and ``arp`` can be used to scan your network, which may also help the core device be discovered.
+
+diagnose and fix sequence errors?
```

Comment:
Would the RTIO analyzer come in handy here?

Reply:
I'm under the impression that the RTIO analyzer only maps slack or lack thereof, which doesn't really have anything to do with sequence errors. "How do I use the analyzer" is probably also a FAQ, or it should be covered in more detail in the tutorial section, but I'd have to add that later whenever I figure out how to use it myself... since a FAQ is so easy to add onto I haven't worried too much about completeness here, just covered what immediately comes to mind.